components/storage/blockdevice/COMPONENT_QSPIF/README.md

```diff
@@ -0,0 +1,85 @@
+# Quad SPI (QSPI) Flash Block Device
+
```

Comment:
Actually, I think, the README should move to the handbook instead of being here at the component.

Reply:
@yossi2le - sure will remove and move to Handbook